src/test/java/uk/gov/pay/directdebit/webhook/gocardless/services/handlers/SendEmailsForGoCardlessEventsHandlerTest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.pay.directdebit.webhook.gocardless.services.handlers;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import uk.gov.pay.directdebit.events.model.GoCardlessEvent;
+import uk.gov.pay.directdebit.events.model.GoCardlessResourceType;
+import uk.gov.pay.directdebit.notifications.services.UserNotificationService;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static uk.gov.pay.directdebit.payments.fixtures.GoCardlessEventFixture.aGoCardlessEventFixture;
+
+@RunWith(JUnitParamsRunner.class)
+public class SendEmailsForGoCardlessEventsHandlerTest {
+
+    @Rule
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
+    
+    @Mock
+    private UserNotificationService userNotificationService;
+    
+    @InjectMocks
+    private SendEmailsForGoCardlessEventsHandler goCardlessMandateHandler;
+
+    @Test
+    @Parameters({"REFUNDS", "SUBSCRIPTIONS"})
```

Comment:
We will also never send emails for payouts

Reply:
fixed